library/Makefile

```diff
@@ -302,6 +302,13 @@ version_features.c:
 	echo "  Gen   $@"
 	$(PERL) ../scripts/generate_features.pl
 
+psa_crypto_driver_wrappers.c: ../scripts/codegen_driverwrappers/generate_driver_wrappers.py
+psa_crypto_driver_wrappers.c:
+	echo "  Gen   $@"
+	$(PYTHON) ../scripts/codegen_driverwrappers/generate_driver_wrappers.py \
```

Comment:
`PYTHON` isn't defined here, so this rule fails on platforms where the shebang line and the extension aren't enough to make the script executable. https://github.com/gilles-peskine-arm/mbedtls/commit/a17150d7acd44c5866a7117f25eca0b030e17a1e